Fix for crash after access to the configured, but not yet paired RFCOMM device.
http://www.spinics.net/lists/linux-bluetooth/msg03299.html
Index: bluez-gnome-1.8/common/bluetooth-client.c
===================================================================
--- bluez-gnome-1.8.orig/common/bluetooth-client.c
+++ bluez-gnome-1.8/common/bluetooth-client.c
@@ -330,7 +330,7 @@ static void add_device(DBusGProxy *adapt
 		name = value ? g_value_get_string(value) : NULL;
 
 		value = g_hash_table_lookup(hash, "Class");
-		type = class_to_type(g_value_get_uint(value));
+		type = value ? class_to_type(g_value_get_uint(value)) : BLUETOOTH_TYPE_ANY;
 
 		value = g_hash_table_lookup(hash, "Icon");
 		icon = value ? g_value_get_string(value) : "bluetooth";
